_cloud/__init__.py

```diff
@@ -9,7 +9,11 @@
 
 
 class Module(object):
-    """The magical API around downloading / installing /Â importing things."""
+    """The magical API around downloading / installing / importing things."""
+    
+    _zip = None
+    metadata = None
```

Comment:
I just want to make sure I can focus at the real warnings, instead of having the editor over saturated with those silly ones.